src/index.js

```diff
@@ -0,0 +1,188 @@
+import './style.css';
+
+import { saveTasks, savedTasks } from './modules/edit.js';
+
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+
+// const completeButton = document.querySelector('.complete');
+
+function removeTask(li) {
+  const index = Array.prototype.indexOf.call(todoList.children, li);
+  savedTasks.splice(index, 1);
+  saveTasks();
+
+  // Update remaining tasks' indexes
```

Comment:
Please make sure you remove unused code from your code. This would allow you to write clean code.